src/index.css

```diff
@@ -81,19 +83,53 @@ button:hover {
   grid-column: span 2;
 }
 
-#root {
+.div-container {
+  padding: 2rem;
+}
+
+.div-container h2 {
+  margin-bottom: 3rem;
+  font-size: 20px;
+}
+
+.div-container p {
+  font-size: 20px;
+  line-height: 3rem;
+}
+
+nav {
+  font-size: 20px;
+  display: flex;
+  height: 50px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 2rem 0 2rem;
+}
+
+nav ul {
+  list-style: none;
   display: flex;
   align-items: center;
-  justify-content: center;
-  gap: 4rem;
+  gap: 2rem;
+}
+
+nav ul li {
+  cursor: pointer;
+  transition: 0.25s ease-out;
+}
+
+nav ul li:hover {
+  text-decoration: underline;
 }
 
 .quotes {
+  margin: 20rem auto;
   border: 1px solid gray;
-  padding: 10px;
+  padding: 100px;
   border-radius: 8px;
   box-shadow: 10px 10px 39px -7px var(--primary);
-  width: 30%;
+  width: 70%;
+  line-height: 4rem;
 }
```

Comment:
- [ OPTIONAL ] On your quotes page, great job getting the quotes from an external API :smile: . But to allow users to properly visualize the beauty of your project, I will suggest that you center the quote properly so it can be viewed. Currently it is stuck beneath the page making it difficult to read its contents so kindly improve upon this :100:
- Your Current Design
![image](https://user-images.githubusercontent.com/102699958/228645271-f474cf4b-2886-41bc-b40e-0d0b82593a0a.png)

Reply:
Hello @NahnahAJ , thank you for the feedback😊. I am working on it ASAP👌